lib/core/src/model.rs

```diff
@@ -643,6 +706,12 @@ pub enum PaymentState {
     /// This covers the case when the swap state is still Created and the swap fails to reach the
     /// Pending state in time. The TimedOut state indicates the lockup tx should never be broadcast.
     TimedOut = 4,
+
+    /// ## Incoming Chain Swaps
+    ///
+    /// This covers the case when the swap failed for any reason and there is a user lockup tx.
+    /// The swap in this case has to be manually refunded with a provided Bitcoin address
+    Refundable = 5,
```

Comment:
There is `refund_outgoing_swap`, so the description of `Refundable` state should mention what it means in the "Outgoing Chain Swaps" case as well.

Reply:
The outgoing chain swaps are automatically refunded back to the L-BTC wallet. If its cooperative it can be done straight away. I guess if it can only be done non-cooperatively, we can mark it Refundable until the locktime expires. Thoughts @roeierez?

Reply:
I don't think so, we are using Refundable to mark these that the user needs to refund manually. If the process is automatic then marking these addresses as refundables would be confusing.
In most cases the refund in these cases is immediate (cooperative) and the payment is still pending untill refund confirms.
I think it makes sense to add a different status, perhaps PENDING_REFUND ? We can use these status immediately after we get a status from the swapper that the payment failed and we know we need to refund.

Reply:
@dangeross @ok300 This PR is already big enough so I suggest to leave current send_swaps as pending and not introducing another status (PENDING_REFUND). This means currently these will be left as PENDING untill they are refunded automatically. I seems reasonable for now IMO and we can discuss that later.

Reply:
Sounds good, then I agree we can leave this as it is, namely `Refundable` only used for receive onchain swaps.